src/components/MagicCodeInput.js

```diff
@@ -142,7 +142,7 @@ function MagicCodeInput(props) {
         props.onFulfill(props.value);
     };
 
-    useOnNetworkReconnect(validateAndSubmit);
+    useNetwork(validateAndSubmit);
```

Comment:
I thought about refactoring this to take an object as an argument for better clarity, so you'd use it like:

```js
useNetwork({onReconnect: validateAndSubmit});
```

I could make that change if you think it would be clearer?

Reply:
@roryabraham yes that looks better (but it's NAB)

Reply:
Updated this because I agree it improves clarity.